django_netjsonconfig/widgets.py

```diff
@@ -27,8 +27,8 @@ def media(self):
     def render(self, name, value, attrs={}):
         attrs['class'] = 'vLargeTextField jsoneditor-raw'
         html = """
-<a href="#advanced_editor" class="button json-editor-btn-edit normal-mode">{0}</a>
```

Comment:
It's invisible but it's what is still used to return to normal mode in the editor. because it is clicked each time the "back to normal mode" button is clicked.